monkestation/code/modules/blacksmithing/finished_items.dm

```diff
@@ -0,0 +1,289 @@
+	overlay_state = "longhandle"
+	reach = 2 //yeah ok
+	slot_flags = ITEM_SLOT_BACK
+	sharpness = IS_SHARP
+
+//////////////////////////
+//      Other Melee     //
+///////////////////////////
+
+
+/obj/item/melee/smith/hammer//blacksmithing, not warhammer.
+	name = "hammer"
+	icon_state = "hammer"
+	overlay_state = "hammerhandle"
+	var/qualitymod = 0
+
+/obj/item/scythe/smithed //we need to inherit scythecode, but that's about it.
+
+
+/obj/item/melee/smith/cogheadclub
+	name = "coghead club"
+	icon_state = "coghead"
+	item_flags = NEEDS_PERMIT
+	overlay_state = "stick"
+
+/obj/item/melee/smith/shortsword
+	name = "gladius"
+	force = 9
+	item_flags = NEEDS_PERMIT
+	sharpness = IS_SHARP
+	icon_state = "gladius"
+	overlay_state = "gladiushilt"
+
+/obj/item/melee/smith/shortsword/scimitar
+	name = "scimitar"
+	sharpness = IS_SHARP
+	icon_state = "scimitar"
+	overlay_state = "scimitarhilt"
+
+/obj/item/melee/smith/wakizashi
+	name = "wakizashi"
+	sharpness = IS_SHARP
+	force = 7
+	item_flags = NEEDS_PERMIT
+	obj_flags = UNIQUE_RENAME
+	icon_state = "waki"
+	overlay_state = "wakihilt"
+
+
+/obj/item/melee/smith/twohand/broadsword
+	name = "broadsword"
+	icon_state = "broadsword"
+	overlay_state = "broadhilt"
+	wielded_mult = 1.8
+
+/obj/item/melee/smith/twohand/zweihander
+	name = "zweihander"
+	icon_state = "zwei"
+	overlay_state = "zweihilt"
+	force = 4
+	wielded_mult = 3 //affected more by quality. a -1 is 25% less damage, a +1 is 25% more. These bonuses are tripled when wielded.
+
+/obj/item/melee/smith/twohand/katana
+	name = "katana"
+	icon_state = "katana"
+	overlay_state = "katanahilt"
+	force = 7
+	wielded_mult = 2
+	item_flags = NEEDS_PERMIT //want to name your katana "DEMON BLADE" or some shit? go ahead, idiot.
+	obj_flags = UNIQUE_RENAME
+
+/obj/item/melee/smith/sabre
+	name = "sabre"
+	icon_state = "sabre"
+	sharpness = IS_SHARP
+	overlay_state = "sabrehilt"
+	armour_penetration = 15
+	force = 9
+	hitsound = 'sound/weapons/rapierhit.ogg'
+	item_flags = NEEDS_PERMIT
+
+/obj/item/melee/smith/sabre/rapier
+	name = "rapier"
+	icon_state = "rapier"
+	sharpness = IS_SHARP
+	overlay_state = "rapierhilt"
+	force = 6 //less force, stronger parry
+	sharpness = IS_SHARP
+	armour_penetration = 30
+
+//unique hammers
+/obj/item/melee/smith/hammer/toolbox
+	name = "toolbox hammer"
+	desc = "A metal filled toolbox on a stick. Useable as a really shitty hammer."
+	w_class = WEIGHT_CLASS_BULKY
+	icon_state = "toolbox"
+	overlay_state = "hammerhandle"
+	qualitymod = -2
+
+/obj/item/melee/smith/hammer/narsie
+	name = "runemetal hammer"
+	custom_materials = list(/datum/material/runedmetal = 12000)
+	desc = "A metal hammer inscribed with geometeric runes."
+	qualitymod = 1
+
+/obj/item/melee/smith/hammer/narsie/attack(mob/living/target, mob/living/carbon/human/user)
+	if(!iscultist(user))
+		user.Paralyze(100)
+		user.dropItemToGround(src, TRUE)
+		user.visible_message("<span class='warning'>A powerful force shoves [user] away from [target]!</span>", \
```

Comment:
span defines

Reply:
unsure what you mean here this is how they are done elsewhere in the code?

Reply:
https://github.com/Monkestation/MonkeStation/pull/316